Add community tools in README.md (#95723)
microsoft/winget-pkgs: README.md

This repository contains the manifest files for the **Windows Package Manager** default source. You are highly encouraged to submit manifests for your favorite application.

> Note: At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.

The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).

# Submitting a Package

To submit a package to this repository, you should follow these steps:

1. Follow the [Contributing](#Contributing) guidelines below.
2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
3. [Test](#test-your-manifest) your manifest.
4. [Submit](#submit-your-pr) your pull request (PR).
5. Respond to any feedback in your PR.

> Note: Please check the package's manifest you intend to submit does not already exist in the repository, and there are no open PRs for it in order to avoid duplicates.

## Authoring a Manifest

> There are some tools available to help you author a manifest.

Reviewer: ```suggestion
A few different tools available to help you author a manifest.
```

- [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create) (**Official**)
- Other WinGet Manifest Creators developed by the community:
  - [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac) (maintained by [**@russellbanks**](https://github.com/russellbanks))

>   - [YamlCreate.ps1](doc/tools/YamlCreate.md)

Reviewer: ```suggestion
  - [YamlCreate.ps1](doc/tools/YamlCreate.md) (maintained by Microsoft & community)
```

> Note: At this time installers must be MSIX, MSI, APPX, or .exe application installers. Script-based installers and fonts are not currently supported.

The **Windows Package Manager** is an [open source client](https://github.com/microsoft/winget-cli) designed for command-line usage. If you are interested in exploring third-party repositories offering private winget package hosting, see [third-party repositories](THIRD_PARTY.md).

# Submitting a Package

To submit a package to this repository, you should follow these steps:

1. Follow the [Contributing](#Contributing) guidelines below.
2. [Author](AUTHORING_MANIFESTS.md) a Manifest.
3. [Test](#test-your-manifest) your manifest.
4. [Submit](#submit-your-pr) your pull request (PR).
5. Respond to any feedback in your PR.

> Note: Please check the package's manifest you intend to submit does not already exist in the repository, and there are no open PRs for it in order to avoid duplicates.

## Authoring a Manifest

There are some tools available to help you author a manifest.

- [Windows Package Manager Manifest Creator](https://github.com/microsoft/winget-create)

> - [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)

Reviewer 2: I had a conversation with @denelon before opening this PR. My thoughts are we should list software in README with what extra functionality/advantages they have above already existing tools. 

We would certainly not add a project to the README that just creates manifests with only mandatory fields with some user input prompts here and there, without any additional metadata. I think we can add a note on it that the tool should stand apart from other tools and should not do the same thing that other tools already do.

> - [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)

Reviewer 3: I agree with @vedantmgoyal2009 in this respect - I've put near enough 1000 hours into writing Komac as a hobby project; I seriously doubt more like it will just crop up by adding it to some documentation here.

> - [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)

Reviewer: I agree. We're leading with "wingetcreate" since it's a product we develop and maintain. It's designed with CI/CD scenarios in mind. YAMLCreate would be next as it's also in the Microsoft organization.

I'd suggest after those something like:

## Alternatives
Other open-source tooling has been developed by third parties. 
* [Option 1](https://url.com)
* [Option 2](https://url.com)

---
I'd rather avoid any specific descriptions for those projects, and I'd list them in alphabetical order. We need to make sure we're not endorsing or otherwise giving any impression of favoritism.

> - [Komac - **Ko**tlin **Ma**nifest **C**reator for WinGet](https://github.com/russellbanks/Komac)

Reviewer: @Trenly I had this same concern when I was chatting with @vedantmgoyal2009. I don't want to create some kind of contest for generating additional tools to get the same job done. I'm more than happy to have some kind of mention or reference to other projects, and clearly the work @russellbanks has done is in another language (so it's not something that would be easy to add to wingetcreate or YAMLCreate) which other developers may prefer, but I don't want to end up with a list of competing projects.

- [YamlCreate.ps1](#using-the-yamlcreateps1)

> Note: Only one manifest may be submitted per PR.

### Using the Komac CLI (maintained by [**@russellbanks**](https://github.com/russellbanks) - a community moderator & contributors)

Download the appropriate installer for your OS from the [releases page](https://github.com/russellbanks/Komac/releases) and install it. Open a terminal and run `komac` to get started.

> - To create a new manifest, run `komac new` and follow the self-explanatory prompts. To give a slight idea, you will be prompted for Package Identifier, Version, and the URLs to the installer. Additionally, you will be prompted for the metadata such as Publisher, Package Name, License, and more. It will take a little effort to fill out all the metadata, but on updating the manifest, you'll be prompted only for Identifier, Version, and Installer URLs.

Reviewer 4: Specific instructions for usage should probably be handled in the KOMAC repository itself or in a dedicated file in /doc

Reviewer: I don't want to have instructions here. I don't want churn because something external changed. It will be enough to keep up with the lists of related packages. Adding manifest creators opens up the door to REST implementations, GUIs, etc.
